packages/builder/src/pages/builder/portal/users/users/index.svelte

```diff
@@ -81,6 +88,14 @@
       width: "1fr",
     },
   }
+
+  // Per user check
+  $: showUserLimitAlert =
+    $admin.cloud &&
+    $licensing.isFreePlan &&
+    $licensing.license.plan.model === Constants.PlanModel.PER_USER &&
+    $licensing.quotaUsage.usageQuota.users > 5
```

Comment:
Can we combine this logic with the logic for the banner in the store, so we can say
```js
$licensing.warnUserLimit
```